source/seedsource/tasks/cleanup_tifs.py

```diff
@@ -0,0 +1,20 @@
+from seedsource_project import settings
+import os
+import os.path
+import time
+import re
+from celery.task import task
+
+@task
+def cleanup_temp_tif_files(age=7200):
+    tempDir = settings.NC_TEMPORARY_FILE_LOCATION
+    cutoff = time.time() - age
+    files = os.listdir(tempDir)
+    for file in files:
+        if re.search('.tif$', file):
+            path = os.path.join(tempDir, file)
+            if os.path.getctime(path) < cutoff:
+                try:
+                    os.remove(path)
+                except:
```

Comment:
This should catch a specific exception, rather than everything. In this case, `OSError` seems most appropriate.